network/p2p/p2p_validation_test.go

```diff
@@ -46,28 +49,103 @@ func TestP2pNetwork_MessageValidation(t *testing.T) {
 	defer cancel()
 
 	// Create 20 fake validator public keys.
-	validators := make([]string, validatorCount)
-	for i := 0; i < validatorCount; i++ {
-		var validator [48]byte
-		cryptorand.Read(validator[:])
-		validators[i] = hex.EncodeToString(validator[:])
-	}
-	var mtx sync.Mutex
+	shares := generateShares(t, validatorCount)
+
 	// Create a MessageValidator to accept/reject/ignore messages according to their role type.
 	const (
 		acceptedRole = spectypes.RoleProposer
 		ignoredRole  = spectypes.RoleAggregator
 		rejectedRole = spectypes.RoleSyncCommitteeContribution
```

Comment:
this tests runs only single validator role messages. we should replace one of the message role to be `RoleCommittee` so we test both types of messages. 

task: Test RoleCommittee messages in `TestP2pNetwork_MessageValidation`